tests/scanners/test.filename.js

```diff
@@ -139,9 +136,9 @@ describe('Hidden and Flagged File Regexes', function() {
 
   for (const filePath of matchingFlaggedFileExtensions) {
     it(`should match ${filePath} as a flagged file extensions`, () => {
-      assert.isOk(
-        constants.FLAGGED_FILE_EXTENSIONS.includes(extname(filePath)),
-        `${filePath} should not match flagged file extension regex`);
+      expect(
+        constants.FLAGGED_FILE_EXTENSIONS.includes(extname(filePath))
```

Comment:
If this used https://facebook.github.io/jest/docs/expect.html#tocontainitem it would give better errors.